src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import cn from "classnames";
+import styles from "./Navbar.module.scss";
+
+export const Navbar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const closeNavbar = () => setIsOpen(false);
+
+  const navItems = [
+    { path: "/menu", text: "Twój jadłospis" },
+    { path: "/recipes", text: "Przepisy" },
+    { path: "/shopping-list", text: "Lista zakupów" },
+    { path: "#", text: "Twój profil" },
+  ];
```

Comment:
`navItems` jest stałą - możemy wynieść to równie dobrze poza komponent. 
```suggestion
const navItems = [
    { path: "/menu", text: "Twój jadłospis" },
    { path: "/recipes", text: "Przepisy" },
    { path: "/shopping-list", text: "Lista zakupów" },
    { path: "#", text: "Twój profil" },
  ];

export const Navbar = () => {
  const [isOpen, setIsOpen] = useState(false);

  const closeNavbar = () => setIsOpen(false);
```
Nie powinno to dla nas dużo zmienić, ale dobrym podejściem jest wyciągać takie stałe poza komponent, bo w przypadku, gdy chcielibyśmy dalej przekazać taką złożoną wartość przez propsy, za każdym re-renderowaniem komponentu - tworzylibyśmy nowy obiekt -> mielibyśmy nową referencję -> mogłoby to wpływać na renderowanie komponentów, do których przekazaliśmy dalej tę wartość. Mając stała - mamy cały czas tę samą referencje do obiektu/tablicy